packages/shared/src/variants/drift.ts

```diff
@@ -1,6 +1,13 @@
 import { Coordinate } from "../lib/coordinate";
+import { Grid } from "../lib/grid";
 import { getGroup } from "../lib/group_utils";
-import { Baduk, BadukConfig, Color, groupHasLiberties } from "./baduk";
+import {
+  Baduk,
+  BadukConfig,
+  Color,
+  groupHasLiberties,
+  isGridBadukConfig,
+} from "./baduk";
 
 export interface DriftGoConfig extends BadukConfig {
```

Comment:
Can this extend `GridBadukConfig` instead?

Reply:
changed to type and extend GridBadukConfig